src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
I believe it's not intuitive to keep the class and file name as Duke as the new chat entity is named Gary. So change it to `Gary.java` and `public class Gary`